app/statistics/server/lib/statistics.js

```diff
@@ -118,6 +122,37 @@ export const statistics = {
 			count,
 		}));
 
+		// Number of departments
```

Comment:
🤔 can we do these queries on a promise.all? don't see any need for them to be sequential